api/spec/controllers/mno_enterprise/impersonate_controller_spec.rb

```diff
@@ -9,18 +9,21 @@ module MnoEnterprise
     let(:user) { build(:user, :admin) }
     let(:user2) { build(:user) }
     before do
-      api_stub_for(get: "/users/#{user.id}", response: from_api(user))
-      api_stub_for(put: "/users/#{user.id}", response: from_api(user))
-      api_stub_for(get: "/users/#{user2.id}", response: from_api(user2))
-      api_stub_for(put: "/users/#{user2.id}", response: from_api(user2))
+      stub_api_v2(:get, "/users/#{user.id}", user, %i(deletion_requests organizations orga_relations dashboards))
+      stub_api_v2(:get, "/users/#{user2.id}", user2, %i(deletion_requests organizations orga_relations dashboards))
+
+      stub_api_v2(:patch, "/users/#{user.id}")
+      stub_api_v2(:patch, "/users/#{user2.id}")
+
     end
+    before { stub_audit_events }
```

Comment:
I don't like the idea to stub it globally as it force the specs to be awayre that the audit_events is being stubbed. 
Also we could improve this stub to test that the right fields and comments are being sent.